public/src/main/java/com/rackspace/salus/telemetry/api/model/TestMonitorAndEventTaskResponse.java

```diff
@@ -16,25 +16,36 @@
 
 package com.rackspace.salus.telemetry.api.model;
 
-import com.rackspace.salus.event.manage.model.TestTaskResult;
-import com.rackspace.salus.monitor_management.web.model.TestMonitorResult;
-import lombok.AllArgsConstructor;
-import lombok.Data;
-
+import com.rackspace.salus.event.manage.model.TestTaskResult.TestTaskResultData.EventResult;
+import com.rackspace.salus.event.model.kapacitor.Task;
+import com.rackspace.salus.telemetry.model.SimpleNameTagValueMetric;
 import java.util.List;
+import lombok.Data;
 
 @Data
-@AllArgsConstructor
 public class TestMonitorAndEventTaskResponse {
 
   private TestMonitorAndEventTask data;
   private List<String> errors;
 
   @Data
-  @AllArgsConstructor
   public static class TestMonitorAndEventTask {
```

Comment:
Sorry, I missed this one before. Would be good to name it with "data" such as
```suggestion
  public static class ResponseData {
```